project/src/components/map/map.tsx

```diff
@@ -1,11 +1,30 @@
+import 'leaflet/dist/leaflet.css';
+import styles from './map.module.css';
 import cn from 'classnames';
+import { City } from '../../types/city';
+import { Point, Points } from 'types/point';
+import { useRef } from 'react';
+import useMap from '../../hooks/useMap';
```

Comment:
Сам файл в кебаб кейсе называем
```suggestion
import useMap from '../../hooks/use-map';
```